### PR TITLE
Update Import-Aliases

### LINKED ISSUE #692: Request preferred alias verification
**What would you like to be added**:
The `Kubernetes` community defined a group of preferred [import aliases](https://github.com/kubernetes/kubernetes/blob/b0836a65f294d879d0000c81faa5d5149f91cf73/hack/.import-aliases), and provide a [verify-import-aliases.sh](https://github.com/kubernetes/kubernetes/blob/b0836a65f294d879d0000c81faa5d5149f91cf73/hack/verify-import-aliases.sh) to verify the codebase.

I think it's a very useful tool to keep code clean and easy to maintain. So, after talking with the tool main author @dims,  I'd like to introduce it to the Karmada project. For the preferred import aliases, we also want to define some aliases specific to the Karmada project, like `v1alpha1work`, `v1alpha1policy`, and so on.

**Why is this needed**:

*Note:* Anybody who interests in this could take the issue and help.

### PR DESCRIPTION
**Pull request by ESonata**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

update import-aliases by running the update-import-aliases.sh

**Which issue(s) this PR fixes**:

Fixes #692 

**Does this PR introduce a user-facing change?**:

```release-note
     NONE
```

### CODE REVIEW

File: pkg/controllers/cluster/cluster_controller.go [hunk 1]
```diff
@@ -6,12 +6,15 @@ import (
 	"sync"
 	"time"
 
-	coordv1 "k8s.io/api/coordination/v1"
-	corev1 "k8s.io/api/core/v1"
+	clusterv1alpha1 "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
```
Comment (XiShanYongYe-Chang) [inline pkg/controllers/cluster/cluster_controller.go:9]: The import packages need to be in the following order:
```
go package

third-party package

karmada package
```

Same as below.

File: cmd/agent/app/agent.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (Garrybest) [inline cmd/agent/app/agent.go:10]: `v1` does not need an alias.

File: pkg/estimator/pb/types.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package pb
 
 import (
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (Garrybest) [inline pkg/estimator/pb/types.go:4]: If modified, the grpc codes and proto files should be re-generated.

File: pkg/apis/cluster/v1alpha1/zz_generated.register.go [hunk 4]
```diff
@@ -3,7 +3,7 @@
 package v1alpha1
 
 import (
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
[the remark this answers is not in the document]
Comment (RainbowMango) [reply pkg/apis/cluster/v1alpha1/zz_generated.register.go:6]: +1

File: pkg/estimator/pb/types.go (hunk 3 above)
Comment (RainbowMango) [reply pkg/estimator/pb/types.go:4]: @Garrybest we need a verification script to check if the proto file is updated to date.

Comment (Garrybest) [reply pkg/estimator/pb/types.go:4]: Sure.

Comment (RainbowMango) [reply pkg/estimator/pb/types.go:4]: Just created #729 to track.

**Comment by RainbowMango** [comment]
Why close this?

**Comment by ESonata** [comment]
I'm not very proficient in some processes of PR. because there are many files involved in the new changes, I don't want to merge directly, so I ran the reset command and returned to the PR before submission, resulting in the closure of the PR.

**Comment by ESonata** [comment]
@RainbowMango

**Comment by RainbowMango** [comment]
No worries, just curious.
You can ask for help next time you face problems, I can share some experiences with you.

**Comment by ESonata** [comment]
OK. @RainbowMango

**Pull request closed without merging**